src/language_concepts/mutability.md

```diff
@@ -43,6 +43,63 @@ fn helper(mut x: i32) {
 }
 ```
 
+### Constants
+
+A constant type is a value that does not change per circuit instance. This is different to a witness which changes per proof. If a constant type that is being used in your program is changed, then your circuit will also change.
```

Comment:
I think this whole section should be reworded, we've been avoiding the use of `const` internally since it can mean several different things. This section seems to introduce `comptime` values so I think it'd be less confusing to call them `comptime` values. We could say something along the lines of "comptime values are values that are known at compile-time" or "evaluated to a constant value at compile-time" or similar.

Reply:
Thanks for the insights.

The "Constants" and "Globals" sections were simply cut-pasted to this page and are intended to be updated with a separate PR. I'll keep the suggestion in mind when I get to it 🙌